src/material-examples/stepper-overview/stepper-overview-example.ts

```diff
@@ -0,0 +1,26 @@
+import {Component} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+
+/**
+ * @title Stepper overview
```

Comment:
Now that you have this example, you can add 
```markdown
<!-- example(stepper-overview) -->
```
To `stepper.md` (under the first sentence summary) and it will embed the example.